packages/terra-badge/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ Changelog
 
 Unreleased
 ----------
+### Removed
```

Comment:
Is this change version correct?

Reply:
I believe so. The terra-mixins package has been removed from terra-badge as it is not used.